components/camel-tika/src/test/java/org/apache/camel/component/tika/TikaParseTest.java

```diff
@@ -0,0 +1,141 @@
+                Map<String, Object> headerMap = exchange.getIn().getHeaders();
+                assertThat(body, instanceOf(String.class));
+                assertThat((String) body, containsString("test"));
+                assertThat(headerMap.get("TikaContent-Type"), equalTo("application/msword"));
+                return true;
+            }
+        });
+        resultEndpoint.assertIsSatisfied();
+    }
+
+    @Test
+    public void testImageParse() throws Exception {
+        File document = new File("src/test/resources/testGIF.gif");
+        template.sendBody("direct:start", document);
+
+        resultEndpoint.setExpectedMessageCount(1);
+
+        resultEndpoint.expectedMessagesMatches(new Predicate() {
+            @Override
+            public boolean matches(Exchange exchange) {
+                Object body = exchange.getIn().getBody(String.class);
+                Map<String, Object> headerMap = exchange.getIn().getHeaders();
+                assertThat(body, instanceOf(String.class));
+                assertThat((String) body, containsString("<body/>"));
+                assertThat(headerMap.get("TikaContent-Type"), equalTo("image/gif"));
+                return true;
+            }
+        });
+        resultEndpoint.assertIsSatisfied();
+    }
+
+    @Test
+    public void testEmptyConfigDocumentParse() throws Exception {
+        File document = new File("src/test/resources/test.doc");
+        template.sendBody("direct:start3", document);
+
+        resultEndpoint.setExpectedMessageCount(1);
+
+        resultEndpoint.expectedMessagesMatches(new Predicate() {
+            @Override
+            public boolean matches(Exchange exchange) {
+                Object body = exchange.getIn().getBody(String.class);
+                Map<String, Object> headerMap = exchange.getIn().getHeaders();
+                assertThat(body, instanceOf(String.class));
+                assertThat((String) body, containsString("<body/>"));
+                assertThat(headerMap.get("TikaContent-Type"), equalTo("application/msword"));
+                return true;
+            }
+        });
+        resultEndpoint.assertIsSatisfied();
+    }
+
+    @Test
+    public void testRegistryConfigDocumentParse() throws Exception {
+        File document = new File("src/test/resources/test.doc");
+        template.sendBody("direct:start3", document);
+
+        resultEndpoint.setExpectedMessageCount(1);
+
+        resultEndpoint.expectedMessagesMatches(new Predicate() {
+            @Override
+            public boolean matches(Exchange exchange) {
+                Object body = exchange.getIn().getBody(String.class);
+                Map<String, Object> headerMap = exchange.getIn().getHeaders();
+                assertThat(body, instanceOf(String.class));
+                assertThat((String) body, containsString("<body/>"));
+                assertThat(headerMap.get("TikaContent-Type"), equalTo("application/msword"));

```

Comment:
I think we should use Content-Type as well as that is a common header, we have it on Exchange also, eg Exchange.CONTENT_TYPE